### PR TITLE
Minor edits to n19825

### PR DESCRIPTION
**Pull request by jnewbery**

### CODE REVIEW

**Comment by jnewbery** [comment]
@dhruv @jonatack

**Review by jonatack — COMMENTED** [review]
Tested ACK comparing localhost rendering with the website on-line.

File: _posts/2021-01-13-#19825.md [hunk 1]
```diff
@@ -12,38 +12,46 @@ commit: e650bba
...
-       peers](https://github.com/bitcoin/bitcoin/blob/86a8b35f/src/net_processing.cpp#L1018)
-       are discouraged over time. Inbound connections to discouraged peers are
-       allowed but preferred for eviction. Outbound connections are not made and
-       their addresses are not gossiped.
-
-- Neither banning nor discouragement protect against DoS attacks as IP
-  addresses are inexpensive.
-
-- Programatic banning or disconnection increase the risk of a network split.
+- `BanMan` has [two functions](https://github.com/bitcoin/bitcoin/blob/86a8b35f/src/banman.h#L27):
```
Comment (jonatack) [inline _posts/2021-01-13-#19825.md:15]:
```suggestion
- The `BanMan` class has [two functions](https://github.com/bitcoin/bitcoin/blob/86a8b35f/src/banman.h#L27):
```

Comment (jonatack) [reply _posts/2021-01-13-#19825.md:15]: (itsm this reads a bit better than just BanMan in code format)

**Comment by jonatack** [comment]
Re-tested ACK. These edits take it to the next level :rocket:

**Pull request merged**